scripts/coverage.sh

```diff
@@ -7,7 +7,7 @@ mkdir -p build-coverage
 cd build-coverage
 cmake -DBUILD_OSTREE=ON -DBUILD_SOTA_TOOLS=ON -DBUILD_DEB=ON -DBUILD_P11=ON -DBUILD_WITH_CODE_COVERAGE=ON -DTEST_PKCS11_ENGINE_PATH="/usr/lib/engines/engine_pkcs11.so" -DTEST_PKCS11_MODULE_PATH="/usr/lib/softhsm/libsofthsm2.so" ../src
 make -j8
-CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=5 make -j6 coverage
+CTEST_OUTPUT_ON_FAILURE=1 CTEST_PARALLEL_LEVEL=3 make -j6 coverage
```

Comment:
What was the problem, and did this help? If Travis was just hanging indefinitely, in my opinion/experience, this doesn't seem to matter.

Reply:
@patrickvacek The error was something like "run out of virtual memory". I'll also have to comment uptane vector tests for now, I'm experiencing transient bugs somewhere in generator.py

Reply:
Sounds annoying. I have seen that before, but I don't remember why anymore, and that is different than what I was thinking.